src/Theme/HeaderTemplate.elm

```diff
@@ -1,20 +1,25 @@
-module Theme.HeaderTemplate exposing (HeaderInfo, view)
+module Theme.HeaderTemplate exposing (view)
 
-import Css exposing (Style, batch)
-import Html.Styled exposing (Html, h1, header, text)
-import Html.Styled.Attributes exposing (css)
+import Css exposing (Style, batch, fontFamilies, fontSize, margin, rem, zero)
+import Html.Styled exposing (Html, a, h6, header, text)
+import Html.Styled.Attributes exposing (css, href)
+import I18n.Keys exposing (Key(..))
+import I18n.Translate exposing (Language, translate)
+import Shared exposing (Model)
 
 
-type alias HeaderInfo =
-    { content : String }
-
-
-view : HeaderInfo -> Html msg
-view headerInfo =
+view : Model -> Html msg
+view model =
+    let
+        t : Key -> String
+        t =
+            translate model.language
+    in
     header [ css [ headerStyle ] ]
-        [ h1
-            []
-            [ text headerInfo.content
+        [ h6 [ css [ headerBrandStyle ] ]
+            [ a [ href "/" ]
+                [ text (t SiteTitle)
+                ]
             ]
```

Comment:
I'd say h1 is a better default, but I do also remember our conversation around making the Guide titles h1's so we may want to pass in the `Route` and toggle on that.

I suspect this issue is a little overlapping with my current #128 - so I'm not sure if we should keep it open or close as a duplicate?